scripts/cmake/Modules/LibAddMacros.cmake

```diff
@@ -572,7 +572,8 @@ function (generate_manpage NAME)
 			add_dependencies (man man-${NAME})
 		endif (RONN_LOC)
 
-		if (INSTALL_DOCUMENTATION)
+		# It is possible that the man page does not exist, if nobody generated it using `ronn`.
+		if (INSTALL_DOCUMENTATION AND EXISTS "${OUTFILE}")
```

Comment:
As discussed: please remove this change but add the rendered man page.

Reply:
I removed the commit, and added the man page in commit b948543.